templates/how-it-works.html

```diff
@@ -154,7 +154,7 @@ <h3 class="p-heading--four u-no-margin--bottom">MAAS is a bare-metal server prov
             <br class="u-hide--medium u-hide--large" />
         </div>
         <div class="col-2 u-vertically-center">
-            <a aria-label="Find out how to install MAAS" href="/docs/installation" class="p-button--positive u-no-margin--bottom">Install MAAS</a>
```

Comment:
This URL already redirects to the 3.1 docs page. Why add a new URL to redirect?

Reply:
It does, but via an expensive to fetch redirect (approximately ~2-3s) - it's the same rationale as the previous comment (suggest we continue it there)